Building native images with quarkus-opentelemetry extension throw a Random/SplittableRandom initialized error at build time

## Describe the bug
QuarkusVersion: Upstream
Extensions: quarkus-opentelemetry, quarkus-opentelemetry-exporter-jaeger
Reproducer: https://github.com/quarkus-qe/beefy-scenarios
Command: `mvn clean verify -Dnative -pl 300-quarkus-vertx-webClient -Pnative -DskipTests`

Error at native build time: 

```
Caused by: com.oracle.graal.pointsto.constraints.UnsupportedFeatureException: Detected an instance of Random/SplittableRandom class in the image heap. Instances created during image generation have cached seed values and don't behave as expected.  To see how this object got instantiated use --trace-object-instantiation=java.util.Random. The object was probably created by a class initializer and is reachable from a static field. You can request class initialization at image runtime by using the option --initialize-at-run-time=<class-name>. Or you can write your own initialization methods and call them explicitly from your main entry point.
Detailed message:
Trace: Object was reached by 
        reading field io.grpc.internal.RetriableStream.random
```

Similar issue ref: https://github.com/quarkusio/quarkus/issues/14904

If we remove open telemetry extension, then the native build works as expected.

--- a/extensions/grpc-common/deployment/src/main/java/io/quarkus/grpc/common/deployment/GrpcCommonProcessor.java
+++ b/extensions/grpc-common/deployment/src/main/java/io/quarkus/grpc/common/deployment/GrpcCommonProcessor.java
@@ -58,7 +58,8 @@ public class GrpcCommonProcessor {
                 // if they were not marked as runtime initialized:
                 .addRuntimeInitializedClass("io.grpc.netty.Utils")
                 .addRuntimeInitializedClass("io.grpc.netty.NettyServerBuilder")
-                .addRuntimeInitializedClass("io.grpc.netty.NettyChannelBuilder");
+                .addRuntimeInitializedClass("io.grpc.netty.NettyChannelBuilder")
+                .addRuntimeInitializedClass("io.grpc.internal.RetriableStream");
         return builder.build();
     }
 

